sources/vega/snapshot/v1/snapshot.proto

```diff
@@ -342,4 +343,12 @@ message RewardsPendingPayouts {
 message RewardPartyAmount {
   string party = 1;
   string amount = 2;
-}
\ No newline at end of file
+}
+
+// FeeState details
+message FeeState {
+  string asset = 1;
+  string maker_fee = 2;
+  string infrastructure_fee = 3;
+  string liquidity_fee = 4;
+}
```

Comment:
I'm not sure exactly, but won't that need the ID of the market the fee engine is instantiated for?

Reply:
Good question. There is no ID included in the [engine struct](https://github.com/vegaprotocol/vega/blob/develop/fee/engine.go#L18).
The way it is used is just being instantiated in the on the market struct. So that should be probably loaded in the context of the market right?